src/app/routes/contact.tsx

```diff
@@ -56,8 +56,8 @@ export async function action({ request }: ActionFunctionArgs) {
                 embed.author = {
                     name: session.login,
                     url: `https://profile.intra.42.fr/users/${session.login}`,
-                    icon_url: session.imageUrl,
```

Comment:
Why remove the icon?

Reply:
Currently it is not stored in the session anymore

Reply:
Oh so we didn't get icons anymore anyway?